packages/manager/src/managers/telemetry/types.ts

```diff
@@ -35,13 +35,13 @@ export const SegmentEventType = {
 	sliceMachine_start: "slice-machine:start",
 	sliceLibrary_beta_modalOpened: "slice-library:beta:modal-opened",
 	sliceLibrary_beta_codeOpened: "slice-library:beta:code-opened",
-	onboarding_step_opened: "onboarding:step-opened",
```

Comment:
❓How can we removed the past event if we keep it for some use cases as a fallback?

Reply:
ok, I need to check that, it shouldn't be used anymore - maybe I missed some place

Reply:
Ok so I don't have all the context, but as far as I understood by talking with Côme, we will keep the old events and also have new ones, as we will have an experiment hook controlling whether the new shared onboarding is visible.

Refactored to preserve the old events and add new ones ✅